examples/car_statem.erl

```diff
@@ -0,0 +1,416 @@
+%%% -*- coding: utf-8 -*-
+%%% -*- erlang-indent-level: 2 -*-
+%%% -------------------------------------------------------------------
+%%% Copyright 2020-     Manolis Papadakis <manopapad@gmail.com>,
+%%%                     Eirini Arvaniti <eirinibob@gmail.com>
+%%%                 and Kostis Sagonas <kostis@cs.ntua.gr>
+%%%
+%%% This file is part of PropEr.
+%%%
+%%% PropEr is free software: you can redistribute it and/or modify
+%%% it under the terms of the GNU General Public License as published by
+%%% the Free Software Foundation, either version 3 of the License, or
+%%% (at your option) any later version.
+%%%
+%%% PropEr is distributed in the hope that it will be useful,
+%%% but WITHOUT ANY WARRANTY; without even the implied warranty of
+%%% MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+%%% GNU General Public License for more details.
+%%%
+%%% You should have received a copy of the GNU General Public License
+%%% along with PropEr.  If not, see <http://www.gnu.org/licenses/>.
+
+%%% @copyright 2020 Spiros Dontas and Kostis Sagonas
+%%% @version {@version}
+%%% @author Spiros Dontas
+
+-module(car_statem).
+-behaviour(gen_server).
+-behaviour(proper_statem).
+
+-include_lib("proper/include/proper.hrl").
+
+
+%% -----------------------------------------------------------------------------
```

Comment:
The relative order of exports and defines differs in this file from that of `car_fsm`.
They should be made consistent (and the `car_fsm` is better).